docs/_style.scss

```diff
@@ -43,6 +43,20 @@ h3 {
 .slick-slider {
   margin: 30px auto 50px;
 }
+.slick-dots {
```

Comment:
Without it the dots are a little bit to the right side of the image:
![image](https://cloud.githubusercontent.com/assets/528677/9903535/c8c3cea0-5c7f-11e5-92fa-42a6efe654f8.png)
With it they are centered:
![image](https://cloud.githubusercontent.com/assets/528677/9903543/dadfe790-5c7f-11e5-8dfb-00c5d9996713.png)